### ISSUE TITLE
Work out what we want to appear in the search summary box

### ISSUE THREAD

**Issue by robredpath** [ThreeSixtyGiving/grantnav#597]
We've left a blank space where the search summary needs to go. 

Agile suggested a very attractive design with an illustration, which they can develop further for us if we want. 

The search summary box serves two purposes for all searches:
* providing inspiration about what can be done with the data
* providing insight into the "shape" of the data - such as the range of time and values that it covers

For well-refined searches, it also provides genuinely helpful aggregations. 

However, there is a significant risk that people take the aggregations at face value when they don't support that, but the search results don't suggest that something is amiss. 

We can help by:
* including copy around the page that helps people to understand what's gone into their search
* providing helpful visualisations as well as textual summaries of the search results 
* linking to Help page(s) that describe the summary in more detail

**Comment by robredpath** [comment]
I've started this out of sequence because I think it will take some back-and-forth early on to get it to the point where we can implement. 

I've put some ideas into a prototype at 
https://threesixtygiving.github.io/grantnav/prototypes/search_results_summary.html

**Comment by robredpath** [comment]
For this sprint, we're going to implement the same summary box as we currently have, with some appropriate copy to warn people about the potential pitfalls. 

We'll speak with agile.coop separately to look at how we can get a new design, and when we've got a better idea of timescales, we can implement their work.

**Comment by Lathrisk** [comment]
Summary boxes also appear on the top of other pages, such as funders, recipients, etc, pages. We might need to decide is this is going to be largely the same summary box that appears on the search page. I am going to assume the same pattern for now.

**Comment by robredpath** [comment]
@Lathrisk there shouldn't be summary boxes on the funder page (as there aren't already) - but there are other tables, if that's what you mean?

**Comment by Lathrisk** [comment]
Ok, there is a `top-stats` box at the top of the funders page that contains the same stats as the summary stats at the top of the search page (with minor differences). I thought the new summary stats box would replace both of these, perhaps incorrectly.

Funders page:
![grantnav threesixtygiving org_funder_GB-COH-RC000766 (1)](https://user-images.githubusercontent.com/517804/108697530-e35b7580-74fa-11eb-9780-694f031cb112.png)


Search page: 
![grantnav-search](https://user-images.githubusercontent.com/517804/108697403-b73ff480-74fa-11eb-8f94-fc5b3d2b1de1.png)

**Comment by robredpath** [comment]
@Lathrisk ooooohhhhh I see, you mean the funder pages (eg http://grantnav.threesixtygiving.org/funder/GB-LAE-GLA ), not the funders page ( http://grantnav.threesixtygiving.org/funders ) 

I think that keeping the existing format for those tables is fine, at least for now.

**Comment by robredpath** [comment]
(SUPER confusing terminology, almost set up to trap the unwary.....)

**Comment by drkane** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Search summary box for GrantNav](https://trello.com/c/ArubaaYP/151-search-summary-box-for-grantnav)

**Comment by drkane** [comment]
Search summary box is looking great. Some feedback from us - we can meet to talk this through if helpful:

## Headings

- "Search results" heading should go under the summary - otherwise 
- should indicate what page you're on (could raise this as another issue)

## Charts:

I would aim for as simple charts as possible - would be good to see what it looks like with just the bars and labels for the bars, and remove all the other parts of the charts.

- remove ability to zoom into chart
- make background the same as box background
- remove gridlines
- remove axis titles
- remove Y axis labels
- make sure all X axis titles are shown (for date awarded)

- clarify that Amount awarded is only GBP

## Right hand side stats:

- change "Total value" to "Total amount"
- change "Total funders" to "Funders
- change "Total recipients" to "Recipients"

## Show highlighted grants

- need link to grant (instead of funder & recipient)
- smallest grant - should be smallest greater than 0
- add year to grant description (particularly for most recent and oldest)
- "Learn more about the filters" should be removed.

## Link to insights

- we're discussing this separately with Michael

**Comment by mariongalley** [comment]
Call with David x2 & Michael:

Have agreed to add a limit to the size of search we'll send to Insights. For now we'll set it to 1,000 and increase it if we can fix the insights memory issue. Please can the Insights call to action be conditional:

- If number of results < 1,000, show current text/link: "To explore this dataset in more detail, open in 360Giving Insights"
- If number of results > 1,000, show alternative copy: "To explore the data in more detail, narrow your search to fewer than 1,000 results to open in 360Giving Insights"

**Comment by michaelwood** [comment]
Dev instance of GN has been updated to the search-summary-box branch http://grantnav.dev.default.threesixtygiving.uk0.bigv.io/search

**Comment by mariongalley** [comment]
### Following fixes have been done:

- Headings: "Search results" heading is under the summary
- Charts: all changes listed above
  - Now that the background matches the overall summary background, I wonder if we should change the axes to black since the white doesn't contrast with the yellow much

### The following are still to do:
- Headings: should indicate what page you're on (can only see this when you're at the bottom of the page)
- Highlighted grants: all of the changes listed above

**Comment by Lathrisk** [comment]
Quick note, the work that Michael has deployed to the dev server is WIP, and there are still a few things to be done. I didn't realise it would be deployed or reviewed at this point. I'll update once the work is complete. I expect that will be tomorrow, hopefully morning.

P.S. One issue might require more work than we currently have time for:
`smallest grant - should be smallest greater than 0`

@kindly is looking into that but if we run out of time we might need to raise this as a separate ticket and come back to it

**Comment by mariongalley** [comment]
Re: Link to Insights, when we do change it to be conditional can we change the wording from 360Giving Insights to 360Insights? I checked with Katie (Comms) and she confirmed that's how we refer to it on our website and in the brand guidelines.

**Comment by mariongalley** [comment]
Understood, thanks @Lathrisk :)

**Comment by Lathrisk** [comment]
> @kindly (David Raznick) is looking into that but if we run out of time we might need to raise this as a separate ticket and come back to it

I've written this up as a separate ticket. We're hoping to deploy the most recent changes to dev in the morning.

**Comment by mariongalley** [comment]
### Call with Jared, David R, Michael and David K:

All changes described in [this comment](https://github.com/ThreeSixtyGiving/grantnav/issues/597#issuecomment-865983763) above are done and look good, except those raised as separate tickets (#760 #761), neither of which is critical to the release.
